.babelrc

```diff
@@ -1,6 +1,11 @@
 {
-    "presets": ["@babel/env"],
+    "presets": ["@babel/preset-env"],
     "plugins": [
+        ["module-resolver",  { "root": ["./"],
+            "alias": {
+                "threeExamples": "./src/Renderer/ThreeExtended/jsPatched/"
```

Comment:
Instead of creating yet another folder, couldn't we take the opportunity to move all files that _really_ extend THREE.js to a proper folder, under `src/` ? Something like `src/ThreeExtended`, not under `Renderer`. The current folder looks like something where we put things we don't know where to put.

Reply:
An even simpler proposal would be to put the file in itowns at the position they would have been in the THREE project.

Reply:
I reorganized the files and folders.

It's difficult to put the patched files in different folders because it would be necessary to specify, at **Git**, all the files to be ignored. And these can't be in `node_modules/three` because `babel-register` doesn't work.